### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Juank628**
## Portfolio: Mobile menu
In this pull request we did the following:
- Added Javascript linter for local enviroment
- Added Javascript linter for github flow
- Added App.js file for Javascript code
- Added Mobile menu

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const mobileMenuIcon = document.getElementById('mobile_menu_icon');
```
Comment (levy002) [inline app.js:1]: - [ ] Kindly make sure that you have the element in your body that has this `id` as a selector. At this moment the `mobile_menu_icon` selector can not be found anywhere in your HTML file which is cause the mobile menu to open once we click on the hamburger icon.

File: index.html [hunk 2]
```diff
@@ -11,6 +11,18 @@
...
 </head>
 <body>
+
+    <header class="mobile-menu no-visible" id="mobile_menu">
+        <nav>
+            <div id="close_menu_icon">X</div>
+            <ul id="mobile_link_list">
+                <li> <a href="#">Portfolio</a> </li>
+                <li> <a href="#">About</a> </li>
+                <li> <a href="#">Contact</a> </li>
```
Comment (levy002) [inline index.html:21]: - [ ] Kindly make sure when one of the links is clicked we directly move to the corresponding part of the page. If the `Contact` link is clicked, we move directly to the contact part of your portfolio.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Juank628  , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Team, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights
- No Linters error ✅
- Descriptive PR ✅
- Great working on the Menu functionality  and design✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**